.github/workflows/pr-team-labels.yml

```
@@ -0,0 +1,15 @@
+name: Assign PR team labels
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  team-labels:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: equitybee/team-label-action@main
```

I *think* it's failing due to specifying `main` as a version here. No idea why it's specified that way in the action's description.
```suggestion
      - uses: equitybee/team-label-action@v1.0.2
```